shampoo/gui_utils.py

```diff
@@ -0,0 +1,141 @@
+"""
+Convenience functions and objects for SHAMPOO's GUI component.
+
+Author: Laurent P. René de Cotret
```

Comment:
I want to ask how you feel about this – don't be offended if you disagree. I haven't put an `author` tag on each of the files in this repository because this git repository has the contribution history logged within each commit. Would you be offended if I scrubbed this line to be consistent with the rest of the package?